src/app/pages/auth/login/login.component.scss

```diff
@@ -0,0 +1,8 @@
+.form{
+  margin-top: 40px;
+  padding: 20px;
+  display: flex;
```

Comment:
why did not use Bootstrap?????

Reply:
because you told me to copy paste the login component and in this project we don't use bootstrap